server/plugin/webhook.go

```diff
@@ -390,6 +390,10 @@ func (p *Plugin) postIssueEvent(event *github.IssuesEvent) {
 			continue
 		}
 
+		if sub.IssueCreation() && action != "opened" {
+			continue
+		}
```

Comment:
Should we disallow setting both `issues` and `issue_creations` when setting up a subscription?

Reply:
Good point :+1:

Reply:
Thanks for the review! Changing it now.